preprocessing-standards.Rmd

```diff
@@ -233,36 +238,49 @@ We perform a t-test to see if there is a significant positive difference between
 samples with high concentrations of standards and samples with low concentration
 
 ```{r}
-# H0: high<=low vs H1: H0^c
-high <- grep("^High",colnames(fmat_pos_watl2)) #c(1,3,6)
-low <- grep("^Low",colnames(fmat_pos_watl2)) #c(2,4,5)
-
-pvalues <- apply(fmat_pos_watl2, 1, function(x) t.test(x[high]-x[low], 
-                        mu = 0, alternative = "greater")$p.value)
-
+# H0: high==low vs H1: H0^c
+high <- grep("^High", colnames(fmat_pos_watl2)) #c(1,3,6)
+low <- grep("^Low", colnames(fmat_pos_watl2)) #c(2,4,5)
+
+pvalues <- apply(fmat_pos_watl2, 1, function(x) {
```

Comment:
two things I changes:
1) I don't think the replocates are paired (although they are named _1, _2 etc), thus, instead of `x[high] - x[low]` I used `x[high], x[low]`.
2) although the one sided test would make sense I changed to a two-sided. Only because of the volcano plot below - and also because it surprised me that we have compounds higher in low.